src/main/modules/helmet/index.ts

```diff
@@ -27,6 +28,7 @@ export class Helmet {
     private setContentSecurityPolicy(app: express.Express): void {
         app.use(
             helmet.contentSecurityPolicy({
+                useDefaults: false,
```

Comment:
Would you be able to try removing the useDefaults. I'm wondering if, now with the formAction below, it works. Just tried it locally and seemed OK but would need to try on the pipeline

Reply:
Removed userDefaults from index.ts